### DISCUSSION TITLE
Welcome to bldr-sfmc Discussions!

### DISCUSSION THREAD

**Discussion by anthonymzupancic** [basetime/bldr-sfmc#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayushthakkur1701** [comment]
@anthonymzupancic Hi .. I am not able to execute the command **bldr config -n**
getting error **if (this.options?.eventHandlers?.onConnectionError) {**

Please give the guidance.

**Reply by ayushthakkur1701** [reply]
I observed that the issue is due to proxy. Even though the environment variables are set for proxy.

**Reply by anthonymzupancic** [reply]
Hey @ayushthakkur1701 let's move this to an issue.

**Reply by anthonymzupancic** [reply]
@ayushthakkur1701 https://github.com/basetime/bldr-sfmc/issues/29

**Reply by ayushthakkur1701** [reply]
I have installed the package as mentioned in guideline. When I execute command to check bldr config -n I face issue **if (this.options?.eventHandlers?.onConnectionError) {** which is when I run it with proxy. 

But the same is working when I run it on my personal laptop which is not having any proxy software running.

**Reply by anthonymzupancic** [reply]
Yeah then my guess is it's something to do with security settings on your machine.

To be honest I'm unsure how to troubleshoot this issue.

You may need to chat with your company's IT/Security team.
